common/audit.cto

```diff
@@ -87,6 +87,15 @@ abstract transaction ClauseEvent extends ContractEvent {
   o String templateUri
 }
 
+transaction ClauseExecuted extends ClauseEvent {
```

Comment:
Why are we using `Executed` here? I thought we had agreed on `Triggered`?

Reply:
hmm, I thought 'ClauseExecuted' was a new event added last night (https://github.com/clauseHQ/svc-integration-workers/commit/640a76585e7775c18d9771077839fd6cd4f91348) <-- should that code in svc-integration-workers be using `ClauseActionCompleted` and `ClauseActionFailed` (which are trigger events)?

Reply:
or should they be `ClauseTriggered` since they have an `event` property? and we can then add `title`, `message`, and `severity` to the`ClauseTriggered` model?